include/net/ethernet.h

```diff
@@ -75,13 +75,26 @@ enum ethernet_hw_caps {
 
 	/** IEEE 802.1AS (gPTP) clock supported */
 	ETHERNET_PTP			= BIT(8),
+
+	/** IEEE 802.1 Qav supported */
```

Comment:
I wish this comment was like one above (for 802.1AS): "IEEE 802.1 Qav (&lt;something>) supported"